README.md

```diff
@@ -1,8 +1,33 @@
 
-# advance-valuation-rulings
+# Advance Valuation Ruling Backend
 
-This is a placeholder README.md for a new repository
+Service providing the public frontend to ARS.
 
-### License
+## Frontend
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+[Advance Valuation Ruling](https://github.com/hmrc/advance-valuation-rulings-frontend)
+
+## Persistence
+This service uses mongodb to persist user answers.
+
+## Requirements
+This service is written in Scala using the Play framework, so needs at least a JRE to run.
+
+JRE/JDK 11 is recommended.
+
+The service also depends on mongodb.
+
+## Running the service
+Using service manager (sm or sm2)
+Use the ARS_ALL profile to bring up all services using the latest tagged releases
+``sm --start ARS_ALL``
+
+run `sm -s` to check what services are running
+###Launching the service locally
+To bring up the service on the configured port 12600, use
+``sbt run``
+## Testing the service
+This service uses sbt-scoverage to provide test coverage reports.
+
+Use the following command to run the tests with coverage and generate a report.
+`sbt clean coverage test it:test coverageReport`
```

Comment:
May be helpful to add the formatting tip below:

## Scalafmt

To prevent formatting failures in a GitHub pull request,
run the below command before pushing to the remote repository.

```
sbt scalafmtAll
```